pipeline/frontend/yaml/types/command.go

```diff
@@ -16,11 +14,7 @@ type Command strslice.StrSlice
 func (s *Command) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	var stringType string
 	if err := unmarshal(&stringType); err == nil {
-		parts, err := shlex.Split(stringType)
-		if err != nil {
-			return err
-		}
-		*s = parts
+		*s = []string{stringType}
```

Comment:
Whats happens to `# ...` commands now? Are they filtered by yaml directly?

Reply:
yes [:-)](https://go.dev/play/p/rTA-Vrj9Zby)

Reply:
-> the yaml nodes do contain them but on serialization they get thrown away

Reply:
... #1032 will delete it :D

meanwhile get rid of the TODO